src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml

```diff
@@ -0,0 +1,101 @@
+<documentation title="Array Indent">
+    <standard>
+    <![CDATA[
+    The opening square bracket/parenthesis of a multi-line array must be indented at least to the same level of the start of the statement.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Opening square bracket/parenthesis token of a multi-line array indented at least to the same level of the start of the statement.">
+        <![CDATA[
+if ($condition) {
+    $a =
+<em>    [</em>
+        1,
+        2,
+    ];
+}
+        ]]>
+        </code>
+        <code title="Invalid: Opening square bracket/parenthesis of a multi-line array not indented at least to the same level of the start of the statement.">
+        <![CDATA[
+if ($condition) {
+    $a =
+<em>[</em>
+        1,
+        2,
+    ];
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The elements of an array must be indented four spaces beyond the indentation level of the opening square bracket/parenthesis.
```

Comment:
Okay, not sure how to rephrase this, but there are some things here which don't ring true/could be confusing:
* The indentation is only checked for the first line of each element (subsequent lines are not checked for indentation).
* "Four spaces beyond"- not sure how much margin that give to interpret this as "at least four spaces", while the rule is "exactly four spaces".
* "the indentation level of the opening square bracket/parenthesis" - it's not about the indentation/position of the brackets, but about the start of statement which includes the array opener.

Let's rethink the phrasing of this rule.

Note: this also means the Valid/Invalid phrasing may need updating.

Reply:
I went with "The first line of each array element must be indented exactly four spaces more than the start of the statement". I also updated the valid/invalid sentences accordingly and changed the `<em>` tags to highlight only the spaces and not the array element. What do you think?